site/src/components/pagination.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Pagination
+description: Pagination related components in the Atlas Design System
+template: standard
+---
+
+# Pagination
+
+The pagination component consists of several elements:
+
+- `pagination-previous` and `pagination-next` for incremental navigation
+- `pagination-list` which displays page items for navigation to individual pages
+- `pagination-link` for page numbers and `pagination-ellipsis` for range numbers
+
+All elements are optional so you can compose your pagination as you wish.
+
+```html
+<nav class="pagination" role="navigation" aria-label="pagination">
+	<a class="pagination-previous" aria-label="previous">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-left"></span>
+		</span>
+	</a>
+	<a class="pagination-next" aria-label="next">
+		<span class="icon" aria-hidden="true">
+			<span class="docon docon-arrow-right"> </span>
+		</span>
+	</a>
+	<ul class="pagination-list">
+		<li>
+			<a class="pagination-link" aria-label="Goto page 1">1</a>
+		</li>
+		<li>
+			<span class="pagination-ellipsis">&hellip;</span>
+		</li>
+		<li>
+			<a class="pagination-link" aria-label="Goto page 45">45</a>
+		</li>
+		<li>
+			<a class="pagination-link is-current" aria-label="Page 46" aria-current="page">46</a>
+		</li>
+		<li>
+			<a class="pagination-link" aria-label="Goto page 47">47</a>
+		</li>
+		<li>
+			<span class="pagination-ellipsis">&hellip;</span>
+		</li>
+		<li>
+			<a class="pagination-link" aria-label="Goto page 86">86</a>
+		</li>
+	</ul>
+</nav>
+```
+
+### Disabling links and specifying page numbers available
+
+You can disable some links if they are not active, or change the amount of page numbers available.
+
+```html
+<nav class="pagination" role="navigation" aria-label="pagination">
+	<a href class="pagination-previous" aria-label="previous" title="This is the first page" disabled>
```

Comment:
I tried clicking on the disabled link, but the click event still triggered and caused a page reload.

Reply:
this seems to be a issue with modern browsers igonoring disables attributes, would prob need to make some behaviors for it to make it not do anything